### ISSUE TITLE
Do not rewrite anchor when url_options is set

### ISSUE THREAD

**Issue by fabn** [Mange/roadie-rails#40]
```ruby
# in my mailer class
include Roadie::Rails::Mailer

# Override url options for this mailer only
def roadie_options
  super.combine(url_options: {host: ENV['ROADIE_URL_HOST']})
end

def my_mailer
  mail_options[:from] = sender
  # ...
  roadie_mail(mail_options)
end
```

```erb
<!-- In erb file -->
<%= link_to t(:top), '#top' -%> <!-- Gives: <a href="#top">Top</a> -->
```
The delivered email contains this html:

```html
<a href="http://roadie-host-url/#top">Top</a>
```

while it should not touch that anchor tag.

**Comment by Mange** [comment]
Good point! I can confirm that this is something that needs to be fixed.
I'll look at it in the following week.

tors 16 apr 2015 19:55 Fabio Napoleoni <notifications@github.com> skrev:

**Comment by Mange** [comment]
I just released `roadie` 3.0.5 which has a fix for this. Thank you for reporting this! :heart:
